content.test.js

```diff
@@ -48,4 +52,83 @@ describe("Content script", () => {
             },
         );
     });
+
+    describe("addMergeLinks", () => {
```

Comment:
This function didn't have any tests previously. I  added two test cases: one for merging with a merge queue and one for merging manually by a user. I included the minimum HTML I could for the function to work, since it's reading from and manipulating the DOM.